src/qt/clientmodel.cpp

```diff
@@ -99,40 +98,21 @@ size_t ClientModel::getMempoolDynamicUsage() const
     return mempool.DynamicMemoryUsage();
 }
 
-double ClientModel::getVerificationProgress() const
+double ClientModel::getVerificationProgress(const CBlockIndex *tipIn) const
 {
-    LOCK(cs_main);
-    return Checkpoints::GuessVerificationProgress(Params().Checkpoints(), chainActive.Tip());
+    CBlockIndex *tip = (CBlockIndex *)tipIn;
```

Comment:
Yeah. Would be nice.. but i guess this would be a too-broad change for this PR scope.